modules/special-rules/pages/armour-1.adoc

```diff
@@ -0,0 +1,9 @@
+= Armour +1
+
+These units have particularly heavy armour -- such as Ork Meganobz.
```

Comment:
```suggestion
These units have particularly heavy armour for their kind, such as Ork Meganobz.
```
I'm not wedded to this idea, but: While on the whole we try to keep rules out of the fluff, I think this suggestion does that while also meshing with how we will add this ability onto a 'base unit'. ('base unit' is my working term, I think it works, but I'm open to other suggestions if you think it is problematic.) 
I've suggested 'kind' here, rather than 'unit', as we don't want the possible rules connotation of the latter here.

Reply:
Hmm. Working from the Firepower +1 text:

> These units are armed with potent weaponry that does not affect their range.

And Assault +1 text:

> These units are particularly fierce melee fighters, without sacrificing their ranged capability -- such as Space Wolves Long Fangs or Grey Hunters.

How about this?
```suggestion
These units have particularly heavy armour, without sacrificing speed or firepower.
```

P.S. For the 'such as' sections I would have defaulted to a comma, but I thought I'd seen existing files that used -- so used that instead. Which do you prefer?